### PR TITLE
[AAP-17392] Add hidden field component

### PR DESCRIPTION
**Pull request by nixocio**
Add hidden field component

### CODE REVIEW

File: framework/PageInputs/HiddenField.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      <InputGroup>
+        <TextInput
+          aria-label={t('hidden value')}
+          placeholder="••••••••••••••••••••••"
+          type="password"
+          autoComplete="off"
+          isDisabled={isValueHidden}
+        />
+        {isValueHidden && (
+          <Button variant="control" onClick={() => onClear()}>
```
Comment (himdel) [inline framework/PageInputs/HiddenField.tsx:74]: clear should not be enabled if ther is no value

Comment (nixocio) [reply framework/PageInputs/HiddenField.tsx:74]: @himdel taking a look. Perhaps, renaming the variable, and making sure we have unit tests.

Comment (nixocio) [reply framework/PageInputs/HiddenField.tsx:70]: @himdel update the logic. And there are unit tests covering both cases.

**Comment by nixocio** [comment]
Perhaps change name since we have PageFormHidden - to be verified.

**Review by marshmalien — APPROVED** [review]
LGTM 👍

**Review by himdel — APPROVED** [review]

**Pull request merged**